typescript/infra/config/environments/testnet4/validators.ts

```diff
@@ -173,72 +173,156 @@ export const validatorChainConfig = (
+      interval: 1,
+      reorgPeriod: getReorgPeriod('superpositiontestnet'),
+      validators: validatorsConfig(
+        {
+          [Contexts.Hyperlane]: ['0x1d3168504b23b73cdf9c27f13bb0a595d7f1a96a'],
+          [Contexts.ReleaseCandidate]: [],
+          [Contexts.Neutron]: [],
+        },
+        'superpositiontestnet',
+      ),
+    },
+    optimismsepolia: {
       interval: 5,
-      reorgPeriod: getReorgPeriod('plumetestnet'),
+      reorgPeriod: getReorgPeriod('optimismsepolia'),
       validators: validatorsConfig(
         {
-          [Contexts.Hyperlane]: [
-            '0xe765a214849f3ecdf00793b97d00422f2d408ea6',
-            '0xb59998f71efc65190a85ac5e81b66bd72a192a3b',
-            '0xc906470a73e6b5aad65a4ceb4acd73e3eaf80e2c',
-          ],
-          [Contexts.ReleaseCandidate]: [
-            '0xe6e6aeecbf7755cdbc50c2683df9f2d100f6399d',
-            '0x27946c13a475233a3b1eb47f0bd0f7cdec3a3983',
-            '0x2596413213368475c96ddfb1ae26666d22093a8b',
-          ],
+          [Contexts.Hyperlane]: ['0x03efe4d0632ee15685d7e8f46dea0a874304aa29'],
+          [Contexts.ReleaseCandidate]: [],
           [Contexts.Neutron]: [],
         },
-        'plumetestnet',
+        'optimismsepolia',
       ),
     },
-    solanatestnet: {
-      interval: 1,
-      reorgPeriod: getReorgPeriod('solanatestnet'),
+    polygonamoy: {
+      interval: 5,
+      reorgPeriod: getReorgPeriod('polygonamoy'),
       validators: validatorsConfig(
         {
-          [Contexts.Hyperlane]: ['0xd4ce8fa138d4e083fc0e480cca0dbfa4f5f30bd5'],
+          [Contexts.Hyperlane]: ['0xf0290b06e446b320bd4e9c4a519420354d7ddccd'],
           [Contexts.ReleaseCandidate]: [],
           [Contexts.Neutron]: [],
         },
-        'solanatestnet',
+        'polygonamoy',
       ),
     },
-    superpositiontestnet: {
-      interval: 1,
-      reorgPeriod: getReorgPeriod('superpositiontestnet'),
+    hyperliquidevmtestnet: {
+      interval: 5,
+      reorgPeriod: getReorgPeriod('hyperliquidevmtestnet'),
       validators: validatorsConfig(
         {
-          [Contexts.Hyperlane]: ['0x1d3168504b23b73cdf9c27f13bb0a595d7f1a96a'],
+          [Contexts.Hyperlane]: ['0xea673a92a23ca319b9d85cc16b248645cd5158da'],
           [Contexts.ReleaseCandidate]: [],
           [Contexts.Neutron]: [],
         },
-        'superpositiontestnet',
+        'hyperliquidevmtestnet',
       ),
     },
-    optimismsepolia: {
+    berabartio: {
       interval: 5,
-      reorgPeriod: getReorgPeriod('optimismsepolia'),
+      reorgPeriod: getReorgPeriod('berabartio'),
       validators: validatorsConfig(
         {
-          [Contexts.Hyperlane]: ['0x03efe4d0632ee15685d7e8f46dea0a874304aa29'],
+          [Contexts.Hyperlane]: ['0x541dd3cb282cf869d72883557badae245b63e1fd'],
           [Contexts.ReleaseCandidate]: [],
           [Contexts.Neutron]: [],
         },
-        'optimismsepolia',
+        'berabartio',
       ),
     },
-    polygonamoy: {
+    // citreatestnet: {
```

Comment:
is this intentionally commented out? i know it's disabled but i see that `hyperliquidevmtestnet` isn't commented out and is also disabled